tools/mk_vendor

```diff
@@ -0,0 +1,854 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+NAME="mk_vendor"
+VERSION="0.1"
+
+USAGE="Usage:
```

Comment:
Would that make sense to extract this to a `tools/README` and link that from the main `README` ?

Reply:
I would rather create a specific documentation in the tlspuffin book.

At some point, `mk_vendor` should be rewritten in rust/python so that we don't have to maintain a big chunk of bash script. Then it will be easier to do link documentations together...

Reply:
Yes, this will obviously be resolved with the future documentation.